packages/composite-checkout-wpcom/src/hooks/wpcom-store.js

```diff
@@ -3,9 +3,10 @@
  */
 import { useRef } from 'react';
 
-export function useWpcomStore( registerStore ) {
+export function useWpcomStore( registerStore, onEvent ) {
 	// Only register once
 	const registerComplete = useRef();
+
```

Comment:
I'd kind of like to keep these statements close together since the comment above applies to the `if` statement also.